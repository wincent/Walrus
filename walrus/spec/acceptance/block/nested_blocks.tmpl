#block body
#block header
[This is the header]
#end

Main text...

#block footer
[This is the footer]
#end
#end