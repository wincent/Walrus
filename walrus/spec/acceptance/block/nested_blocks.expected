[This is the header]

Main text...

[This is the footer]
